src/object-number.scss

```diff
@@ -0,0 +1,63 @@
+@import "./new-settings";
+@import "./mixins";
+
+/*!
+.fd-object-number+(--success | --warning | --error)
+*/
+$block: #{$fd-namespace}-object-number;
+
+$color-states: (
+  "positive": ("color": var(--sapPositiveTextColor)),
+  "critical": ("color": var(--sapCriticalTextColor)),
+  "negative": ("color": var(--sapNegativeTextColor)),
+  "informative": ("color": var(--sapInformativeTextColor))
+);
+
+.#{$block} {
+  // TEXT VARIABLES
+  $fd-object-number-font-family: var(--sapFontFamily) !default;
+  $fd-object-number-text-font-size: var(--sapFontMediumSize) !default;
+  $fd-object-number-unit-font-size: var(--sapFontSmallSize) !default;
+  $fd-object-number-neutral: var(--sapNeutralTextColor);
+
+  // LARGE DESIGN VARIABLES
+  $fd-object-number-font-size-large: 1.5rem;
+
+  @include fd-reset();
+
+  font-family: $fd-object-number-font-family;
+  color: $fd-object-number-neutral;
+  max-width: inherit;
+
+  &__text,
+  &__unit {
+    max-width: inherit;
+    display: inline-block;
+    vertical-align: middle;
+    padding-top: 0.125rem;
+
+    @include fd-ellipsis();
+  }
+
+  &__text {
+    font-size: $fd-object-number-text-font-size;
+  }
+
+  &__unit {
+    font-size: $fd-object-number-unit-font-size;
```

Comment:
The unit text has different font-family:  `font-family: var(--sapFontLightFamily);`

Reply:
![Screen Shot 2020-08-24 at 9 59 12 AM](https://user-images.githubusercontent.com/50607147/91053210-808f2d00-e5f0-11ea-9c78-f75d05a436cc.png)
Different font families

Reply:
@stefanoScalzo thanks! Could you please re-attach pdf-file with correct spec [here](https://app.gitbook.com/@fundamental-styles/s/docs/new-component-object-number)?

Reply:
@olshansky I updated the pdf in GitBook